lib/puppet/provider/cs_location/crm.rb

```diff
@@ -18,7 +18,12 @@ def self.instances
     instances = []
 
     cmd = [ command(:crm), 'configure', 'show', 'xml' ]
-    raw, status = Puppet::Util::SUIDManager.run_and_capture(cmd)
+    if Puppet::PUPPETVERSION.to_f < 3.4
```

Comment:
That's an "interesting" way to perform version comparison.

Let's please not go there.